src/device-registry/utils/create-airqloud.js

```diff
@@ -502,18 +501,21 @@ const createAirqloud = {
 
         if (responseFromListSites.success === true) {
           const sites = responseFromListSites.data;
-          logObject("sites", sites);
           for (const site of sites) {
-            const { latitude, longitude } = site;
-
-            const isSiteInAirQloud = geolib.isPointInPolygon(
-              { latitude, longitude },
-              airqloudPolygon
-            );
-
-            if (isSiteInAirQloud === true) {
-              site_ids.push(site._id);
-            } else if (isSiteInAirQloud === false) {
+            const { latitude, longitude, _id, description } = site;
+            if (!isEmpty(latitude) && !isEmpty(longitude)) {
+              const isSiteInAirQloud = geolib.isPointInPolygon(
+                { latitude, longitude },
+                airqloudPolygon
+              );
+              if (isSiteInAirQloud === true) {
+                site_ids.push(site._id);
+              } else if (isSiteInAirQloud === false) {
+              }
```

Comment:
Hi @Baalmart this` if else if` conditional statement looks incomplete
1. What happens if isSiteInAirQloud is neither true or false
2. Why is the else if clause empty

Reply:
Thanks @NoahNsimbe for this observation.
There is no serious effect, the rest of the alternative flows can just be logged accordingly.

Let me just raise a quick hotfix for **just logging details** of any Site which does not belong to any AirQloud.

Thanks again!